.github/workflows/latest_dependency_checker.yml

```diff
@@ -20,8 +20,9 @@ jobs:
         python -m pip install -e . 
         python -m pip install -r test-requirements.txt
         make checkdeps OUTPUT_PATH=featuretools/tests/requirement_files/latest_requirements.txt
-    - name: Create Pull Request
-      uses: FeatureLabs/create-pull-request@v3
```

Comment:
I think we had originally used our fork of this action with the idea that changes outside our control wouldn't break our checker. Not sure if this is much of a risk or not?

Reply:
We've had that worry for a while and nothing has happened. If it breaks, it breaks (which I think is highly unlikely).